share/lib/python/neuron/rxdtests/tests/IraHH.py

```diff
@@ -0,0 +1,237 @@
+gliapump = (1.0/3.0) * (p / (1.0 + exp((25.0 - gnai) / 3.0))) * (1.0 / (1.0 + exp(3.5 - ko)))
+g_glia = g_gliamax / (1.0 + exp(-oa_bath*alpha - 2.5)/0.2)
+glia12 = (g_glia / 1.0 + exp((18.0 - ko)/2.5))
+
+#RATES--------------------------------------------------------------------------------------------------------------------------
+
+#m_gate = rxd.Rate(mgate, (minf - mgate)/mtau)
+#h_gate = rxd.Rate(hgate, (hinf - hgate)/htau)
+#n_gate = rxd.Rate(ngate, (ninf - ngate)/ntau)
+
+#dm/dt
+m_gate = rxd.Rate(mgate, (alpha_m * (1.0 - mgate)) - (beta_m * mgate))
+
+#dh/dt
+h_gate = rxd.Rate(hgate, (alpha_h * (1.0 - hgate)) - (beta_h * hgate))
+
+#dn/dt
+n_gate = rxd.Rate(ngate, (alpha_n * (1.0 - ngate)) - (beta_n * ngate))
+
+#d[O2]o/dt
+o2_ecs = rxd.Rate(o2ecs, (epsilon * (oa_bath - o2ecs))) 
+
+#CURRENTS/LEAKS ----------------------------------------------------------------------------------------------------------------
+
+#sodium (Na) current
+na_current = rxd.MultiCompartmentReaction(nai, nao, gna*(v - ena), mass_action=False, membrane=mem, membrane_flux=True)
+
+#potassium (K) current
+k_current = rxd.MultiCompartmentReaction(ki, ko, gk*(v - ek), mass_action=False, membrane=mem, membrane_flux=True)
+
+#chlorine (Cl) current
+cl_current = rxd.MultiCompartmentReaction(cli, clo, gclbar_l * (v - ecl), mass_action=False, membrane=mem, membrane_flux=True)
+
+#nkcc1 (Na+/K+/2Cl- cotransporter)
+nkcc1_current = rxd.MultiCompartmentReaction(2*clo + ko + nao, 2*cli + ki + nai, nkcc1 *volume_scale, mass_action=False, membrane=mem, membrane_flux=True)
+
+#kcc2 (K+/Cl- cotransporter)
+kcc2_current = rxd.MultiCompartmentReaction(clo + ko, cli + ki, kcc2 *volume_scale, membrane=mem, mass_action=False, membrane_flux=True)
+
+#sodium leak
+na_leak = rxd.MultiCompartmentReaction(nai, nao, gnabar_l*(v - ena), mass_action=False, membrane=mem, membrane_flux=True)
+
+#potassium leak
+k_leak = rxd.MultiCompartmentReaction(ki, ko, gkbar_l*(v - ek), mass_action=False, membrane=mem, membrane_flux=True)
+
+#Na+/K+ pump current in neuron (2K+ in, 3Na+ out)
+pump_current = rxd.MultiCompartmentReaction(ki, ko, -2.0*pump*volume_scale, mass_action=False, membrane=mem, membrane_flux=True) 
+pump_current_other_half = rxd.MultiCompartmentReaction(nai, nao, 3.0*pump*volume_scale, mass_action=False, membrane=mem, membrane_flux=True) 
+oxygen = rxd.MultiCompartmentReaction(o2ecs[ecs], dump[cyt], pump *volume_scale, mass_action=False, membrane=mem)
+goxygen = rxd.MultiCompartmentReaction(o2ecs[ecs], dump[gcyt], gliapump *glia_volume_scale, mass_action=False, membrane=gmem)
+ 
+#Na+/K+ pump current in glia (2K+ in, 3Na+ out)
+gliapump_current = rxd.MultiCompartmentReaction(2*gki, 2*ko, -gliapump *glia_volume_scale, mass_action=False, membrane=gmem, membrane_flux=True) 
+gliapump_current_other_half = rxd.MultiCompartmentReaction(3*gnai, 3*nao, gliapump *glia_volume_scale, mass_action=False, membrane=gmem, membrane_flux=True) 
+
+
+# In[8]:
+
+
+stimA = h.IClamp(soma(0.5))
```

Comment:
Make these tests more modular with import guards
```python
if __name__ == "__main__":
    h = sim_init()
    h.continuerun(100)
```

Reply:
These tests will move into the pytest driven testing framework in subsequent PRs.